### DISCUSSION TITLE
Laravel Validation Rule Required_Without

### DISCUSSION THREAD

**Discussion by jimoheokoh** [laravel/framework#38776] [Q&A]
- Laravel Version: 8.40
- PHP Version: 7.4

### Description:
I am trying to use Laravel Validation Rule Required_without
https://laravel.com/docs/8.x/validation#rule-required-without

![image](https://user-images.githubusercontent.com/22915817/132993775-c8aab4ed-61de-45a4-99d2-46f5f5c31f29.png)

however on testing the request, I get
![image](https://user-images.githubusercontent.com/22915817/132993828-c39e739c-b25d-4f57-bcf0-60c11a9fe7b9.png)

according to laravel docs
![image](https://user-images.githubusercontent.com/22915817/132993871-73f6b6c7-7d2e-40e0-95a6-e565e15964d0.png)

so since status is present in my request, the qty field should not be required.
even if I end up removing the status request, same validation error.

**Comment by Jubeki** [comment]
It works as intended: https://github.com/Jubeki/laravel-bug-report-38775
You can call the route `/test` and see that the validation fails and then change the value in `routes/web.php` to see that the validation passes.

Maybe this has something todo with your rule notation or another problem in your app.

**Comment by jimoheokoh** [comment]
> It works as intended: https://github.com/Jubeki/laravel-bug-report-38775
> You can call the route `/test` and see that the validation fails and then change the value in `routes/web.php` to see that the validation passes.
> 
> Maybe this has something todo with your rule notation or another problem in your app.

I have replicated your test and see that this validation works for status present in the validation rules. If status not present in validation rule it fails, if present it passes.

But what I was really talking about is status present on the HTTP Request. If the status is present in the HTTP request, it fails
I guess, that's not this Rule Required_without is for, If so, I will find a workaround for it

Thanks

**Comment by jimoheokoh** [comment]
I figured out what the problem was for POST request.
Maybe you guys might want to look at it too..

It fails when you leave a space between like I did above with 
`'qty' => 'required_without_status | numeric'`
but worked with
`'qty' => 'required_without_status| numeric'`
weird!

For PUT request, it fails totally! @Jubeki please look at this again
